middleware/2023-02-15-1.3.43.md

```diff
@@ -0,0 +1,40 @@
+---
+authors: poscreator
+slug: middleware/1.3.43
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.43 (Germany)
+In this version of the Middleware we solved a bug concerning a sorting error in our DSFinV-K export. Additionally, we fixed a bug by changing back slash to slash in the path for FCC used on Linux.
```

Comment:
```suggestion
In this version of the Middleware we solved a bug concerning a sorting error in our DSFinV-K export. Additionally, we fixed a bug by changing backslash to forward slash in the path for FCC used on Linux.
```

Sorry, I was curious and fell into a small rabbithole about the spelling of slashes 😅
https://english.stackexchange.com/questions/289633/why-is-forward-slash-not-spelled-forwardslash#289639